tools/DebianPackageTool/package_tool

```diff
@@ -152,6 +152,9 @@ package_all(){
     package_absolute_placement
     package_samples
     package_docs
+    if [[ $PACKAGE_NAME == "dotnet-dev-"* ]]; then
+        package_debian
```

Comment:
We shouldn't be leaking semantic knowledge of the cli into the `package_tool` like this, it is intended to be general purpose. 

Why not remove the guard altogether and calling this function always?